prompb/types.proto

```diff
@@ -18,6 +18,25 @@ option go_package = "prompb";
 
 import "gogoproto/gogo.proto";
 
+message MetricMetadata {
+  enum MetricType {
+    Unknown        = 0;
+    Counter        = 1;
+    Gauge          = 2;
+    Histogram      = 3;
+    GaugeHistogram = 4;
+    Summary        = 5;
+    Info           = 6;
+    StateSet       = 7;
+  }
+
+  // Represents the metric type, these match the set from Prometheus.
```

Comment:
```suggestion
  // Represents the metric type, these has to match the set from Prometheus.
```
Can we also provide link to exact file where it is?

Reply:
I have added the link to the file, the English form, however, is correct as is.

Reply:
xd 

thanks!